api/v3/Activity.php

```diff
@@ -470,10 +470,16 @@ function _civicrm_api3_activity_get_formatResult($params, $activities, $options)
         break;
 
       case 'case_id':
-        $dao = CRM_Core_DAO::executeQuery("SELECT activity_id, case_id FROM civicrm_case_activity WHERE activity_id IN (%1)",
+      case 'case_subject':
+        $dao = CRM_Core_DAO::executeQuery("SELECT activity_id, case_id, subject FROM civicrm_case_activity cca INNER JOIN civicrm_case cc ON cc.id = cca.case_id WHERE activity_id IN (%1)",
```

Comment:
This is covered in the test isn't it? It feels a bit fragile but as long as the test covers it...